src/components/pages/Splash/index.tsx

```diff
@@ -29,8 +29,8 @@ export default function Splash() {
                 <Space size="large" className={styles.buttonspace}>
                   <Button
                     shape="round"
-                    size="large"
```

Comment:
This shouldnt be removed as its recommended by the library and helps with scaling. Any "main focus" button should have this attribute